src/parser/csql_grammar.y

```diff
@@ -5405,9 +5448,21 @@ only_all_class_spec
 		DBG_PRINT}}
 	;
 
+class_name_with_server_name
+        : class_name '@' dblink_server_name
+                {{DBG_TRACE_GRAMMAR(class_name_with_server_name, : class_name '@' dblink_server_name);
+                assert($1 != NULL && $3 != NULL);
+                container_2 ctn;
+                // Do NOT automatically assign a user name. 
+                PT_NAME_INFO_CLEAR_FLAG($1, PT_NAME_INFO_USER_SPECIFIED);
```

Comment:
Since the target_owner of _db_synonym is NOT NULL, the schema name of the current session must be attached.
So the PT_NAME_INFO_USER_SPECIFIED flag must not be removed.

How about attaching the schema name of the current session, and removing it when using it?

Reply:
This is removed from next commit.